backend/src/routers/gameRouter.ts

```diff
@@ -267,16 +270,60 @@ gameRouter.delete('/:id', async (req, res) => {
  *    "message": "Game marked as played"
  *   }
  *
+ * @apiErrorExample {json} Unauthorized:
+ * HTTP/1.1 401 Unauthorized
+ *   {
+ *    "message": "Unauthorized"
+ *   }
+ *
  * @apiUse ZodError
  */
-gameRouter.post('/markPlayed', async (req, res) => {
+gameRouter.post('/markPlayed/:gameId', async (req, res) => {
 	try {
-		if (!req.user) return res.status(401).json({ message: 'Unauthorized' });
-		await markGameAsPlayed(req.body.gameId, (req.user as GammaUser).cid);
+		if (!req.isAuthenticated()) return res.status(401).json({ message: 'Unauthorized' });
+		await markGameAsPlayed(req.params.gameId, (req.user as GammaUser).cid);
 		res.status(200).json({ message: 'Game marked as played' });
 	} catch (e) {
-		if (e instanceof Error) res.status(400).json({ message: e.message });
-		else res.status(400).json({ message: 'Error marking game as played' });
+		if (e instanceof Error)
+			res.status(500).json({ message: e.message });
+		else
+			res.status(500).json({ message: 'Error marking game as played' });
+	}
```

Comment:
When this gets merged, this will pretty much be done automatically. You should probably remove the overarching try catch